cli/medperf/config.py

```diff
@@ -1,16 +1,13 @@
 from ._version import __version__
 from pathlib import Path
 
-BASE_DIR = Path(__file__).resolve().parent.parent.parent
```

Comment:
`BASE_DIR` has no sense when client is installed in standard way (as it points somewhere to `/your/path/to/python/env/lib/site-packages/`)